docs/source/models/intervention_models/intrapartum/misoprostol_intervention.rst

```diff
@@ -0,0 +1,274 @@
+      and birthing parents should have access to more effective injectible uterotonics. [Tunçalp-2012-Cochrane-Review]_
+
+.. todo:: 
+
+  Replace placeholder baseline coverage with data-backed estimate once we find a better data source. 
+
+Vivarium Modeling Strategy
+--------------------------
+
+This intervention requires adding an attribute to all simulants who attended ANC facilities during their pregnancy and give birth at home to specify if a pregnant person 
+receives misoprostol during labor or not.  We will track this and the model will have different incidence rates for maternal hemorrhage for individuals with and without 
+misoprostol (implemented with a slightly confusing application of our ``Risk`` and ``RiskEffect`` components from ``vivarium_public_health``).
+
+The ``Risk`` component adds an attribute to each simulant indicating whether the simulant has received misoprostol during the intrapartum period. Only simulants who attended ANC
+during pregnancy and who give birth at home are eligible for this intervention.
+
+To make this work naturally with the ``RiskEffect`` component, it is best to think of the risk as "no misoprostol".  With this framing, the ``RiskEffect`` 
+component requires data on (1) the relative risk of maternal hemorrhage incidence for people who did not receive misoprostol before labor began, and (2) the population attributable fraction (PAF) of maternal hemorrhage 
+due to not receiving misoprostol.  We will use the decision tree below to estimate the probability of maternal hemorrhage incidence with and without the use of misoprostol, ensuring consistency
+with the baseline delivery facility rates and baseline misoprostol coverage.
+
+In Vivarium, this risk effect will modify the maternal hemorrhage incidence pipeline, resulting in 
+
+.. math::
+
+   \text{IR}_i^\text{maternal hemorrhage} = \text{IR}^\text{maternal hemorrhage}_ \cdot (1 - \text{PAF}_\text{no misoprostol}) \cdot \text{RR}_i^\text{no misoprostol}
+
+where :math:`\text{RR}_i^\text{no misoprostol}` is simulant *i*'s individual relative risk for "no misoprostol", meaning :math:`\text{RR}_i^\text{no misoprostol} = \text{RR}_\text{no misoprostol}` 
+if simulant *i* does not receive misoprostol, and :math:`\text{RR}_i^\text{no misoprostol} = 1` if simulant *i* receives misoprostol. 
+
+The relative risk value we will use is pulled from [Tunçalp-2012-Cochrane-Review]_, the most recent Cochrane Review of the effect of 
+sublingually received misoprostol during labor on the prevention of maternal hemorrhage.
+
+.. list-table:: Risk Effect Parameters for No Misoprostol
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Parameter
+    - Mean
+    - Distribution
+    - Notes
+  * - Relative Risk
+    - 1.52
+    - :math:`\text{Normal}(1.52,0.08^2)`
+    - Based on relative risk of 0.66 (95% CI 0.10-0.94) on maternal hemorrhage incidence for pregnant people receiving misoprostol
```

Comment:
Similar to azithromycin, let's update this uncertainty interval too https://github.com/ihmeuw/vivarium_research/pull/1673